src/Tools/ExternalAccess/FSharp/NavigateTo/FSharpNavigateToItemKind.cs

```diff
@@ -11,6 +11,7 @@ internal static class FSharpNavigateToItemKind
         public static string Line => NavigateToItemKind.Line;
         public static string File = NavigateToItemKind.File;
         public static string Class => NavigateToItemKind.Class;
+        public static string Record => NavigateToItemKind.Record;
```

Comment:
This class is used in dotnet/fsharp repository. I think we need to check with the other team whether they want this or not?

Reply:
this is fine.